### ISSUE TITLE
Cannot play vimeo

### ISSUE THREAD

**Issue by parkmino** [gnome-mpv/gnome-mpv#323]
Overview Description: Cannot play vimeo video.

Steps to Reproduce:
1. Open Location > Location : e.g. https://www.vimeo.com/240640253
2.
3.

Actual Results: Error message shows up

Expected Results: Play vimeo video

Version: 0.13

Additional Information: mpv plays it flawlessly

**Comment by gnome-mpv** [comment]
Cannot reproduce.

What is the error message that you're getting?

**Comment by parkmino** [comment]
'Playback was terminated abnormally. Reason: no audio or video played.'

**Comment by gnome-mpv** [comment]
Start `gnome-mpv` from CLI like this `G_MESSAGES_DEBUG=all gnome-mpv`, open the video again, and post the log messages here.

**Comment by parkmino** [comment]
`(gnome-mpv:2062): Gtk-DEBUG: Connecting to session manager
** INFO: Starting GNOME MPV 0.13
** INFO: Failed to retrieve property "idle-active" using mpv format 3. Reason: core not uninitialized.
** (gnome-mpv:2062): DEBUG: Applying default option --vo=opengl,vdpau,vaapi,xv,x11,opengl-cb,
** (gnome-mpv:2062): DEBUG: Applying default option --osd-level=1
** (gnome-mpv:2062): DEBUG: Applying default option --softvol=yes
** (gnome-mpv:2062): DEBUG: Applying default option --force-window=immediate
** (gnome-mpv:2062): DEBUG: Applying default option --input-default-bindings=yes
** (gnome-mpv:2062): DEBUG: Applying default option --audio-client-name=gnome-mpv
** (gnome-mpv:2062): DEBUG: Applying default option --title=${media-title}
** (gnome-mpv:2062): DEBUG: Applying default option --autofit-larger=75%
** (gnome-mpv:2062): DEBUG: Applying default option --window-scale=1
** (gnome-mpv:2062): DEBUG: Applying default option --pause=yes
** (gnome-mpv:2062): DEBUG: Applying default option --ytdl=yes
** (gnome-mpv:2062): DEBUG: Applying default option --load-scripts=no
** (gnome-mpv:2062): DEBUG: Applying default option --osd-bar=no
** (gnome-mpv:2062): DEBUG: Applying default option --input-cursor=no
** (gnome-mpv:2062): DEBUG: Applying default option --cursor-autohide=no
** (gnome-mpv:2062): DEBUG: Applying default option --softvol-max=100
** (gnome-mpv:2062): DEBUG: Applying default option --config=no
** (gnome-mpv:2062): DEBUG: Applying default option --config-dir=/home/parkmino/.config/gnome-mpv
** (gnome-mpv:2062): DEBUG: Applying default option --watch-later-directory=/home/parkmino/.config/gnome-mpv/watch_later
** (gnome-mpv:2062): DEBUG: Applying default option --screenshot-template=gnome-mpv-shot%n
** INFO: Loading config file: /home/parkmino/.config/mpv/mpv.conf
** (gnome-mpv:2062): DEBUG: Using temporary input config file: /tmp/.ZPSP8Y
** (gnome-mpv:2062): DEBUG: Applying extra mpv options: 
** (gnome-mpv:2062): DEBUG: Attaching mpv window to wid 0x220003a
** INFO: Using mpv 0.26.0
** (gnome-mpv:2062): DEBUG: Setting volume to 100.000000
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** (gnome-mpv:2062): DEBUG: Retrieved option --window-scale=1.000000
** (gnome-mpv:2062): DEBUG: Retrieved option --autofit-larger=75%x0
** (gnome-mpv:2062): DEBUG: Retrieved option --window-scale=1.000000
** (gnome-mpv:2062): DEBUG: Retrieved option --autofit-larger=75%x0
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "aid"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "vid"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "sid"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "core-idle"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "idle-active"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "fullscreen"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "pause"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "loop"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "playlist"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "playlist-count"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "speed"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "track-list"
** INFO: Loading file (append=FALSE): https://www.vimeo.com/240640253
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "vo-configured"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "idle-active"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "pause"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "media-title"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "playlist"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "playlist-count"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "playlist-pos"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "volume"
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** INFO: Failed to retrieve property "time-pos" using mpv format 5. Reason: property unavailable.
** (gnome-mpv:2062): DEBUG: Sent global key up to mpv
** (gnome-mpv:2062): DEBUG: Retrieved option --window-scale=1.000000
** (gnome-mpv:2062): DEBUG: Retrieved option --autofit-larger=75%x0
** (gnome-mpv:2062): DEBUG: Retrieved option --window-scale=1.000000
** (gnome-mpv:2062): DEBUG: Retrieved option --autofit-larger=75%x0
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "idle-active"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "pause"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "media-title"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "playlist"
** (gnome-mpv:2062): DEBUG: Received mpv property change event for "playlist-pos"
** (gnome-mpv:2062): DEBUG: Set mouse location to (284, 229)
** (gnome-mpv:2062): DEBUG: Set mouse location to (282, 231)
** (gnome-mpv:2062): DEBUG: Set mouse location to (280, 233)
** (gnome-mpv:2062): DEBUG: Set mouse location to (279, 234)
** (gnome-mpv:2062): DEBUG: Set mouse location to (278, 235)
** (gnome-mpv:2062): DEBUG: Set mouse location to (278, 236)
** (gnome-mpv:2062): DEBUG: Set mouse location to (277, 238)
** (gnome-mpv:2062): DEBUG: Set mouse location to (277, 240)
** (gnome-mpv:2062): DEBUG: Set mouse location to (277, 241)
** (gnome-mpv:2062): DEBUG: Set mouse location to (276, 243)
** (gnome-mpv:2062): DEBUG: Set mouse location to (276, 244)
** (gnome-mpv:2062): DEBUG: Set mouse location to (276, 245)
** (gnome-mpv:2062): DEBUG: Set mouse location to (276, 247)
** (gnome-mpv:2062): DEBUG: Set mouse location to (276, 248)
** (gnome-mpv:2062): DEBUG: Set mouse location to (279, 249)
** (gnome-mpv:2062): DEBUG: Set mouse location to (286, 249)
** (gnome-mpv:2062): DEBUG: Set mouse location to (303, 249)
** (gnome-mpv:2062): DEBUG: Set mouse location to (322, 247)
** (gnome-mpv:2062): DEBUG: Set mouse location to (345, 240)
** (gnome-mpv:2062): DEBUG: Set mouse location to (372, 233)
** (gnome-mpv:2062): DEBUG: Set mouse location to (405, 224)
** (gnome-mpv:2062): DEBUG: Set mouse location to (438, 215)
** (gnome-mpv:2062): DEBUG: Set mouse location to (471, 204)
** (gnome-mpv:2062): DEBUG: Set mouse location to (504, 189)
** (gnome-mpv:2062): DEBUG: Set mouse location to (537, 174)
** (gnome-mpv:2062): DEBUG: Set mouse location to (570, 161)
** (gnome-mpv:2062): DEBUG: Set mouse location to (597, 148)
** INFO: Terminating mpv`

**Comment by gnome-mpv** [comment]
Can you put `--msg-level=all=v` in `Extra MPV Options` and post the logs again?

**Comment by parkmino** [comment]
Here you go ;-)

[log.txt](https://github.com/gnome-mpv/gnome-mpv/files/1436132/log.txt)

**Comment by gnome-mpv** [comment]
Compared to the log from a successful playback on my machine, `ffmpeg` is being passed a slightly different URL.

Can you post a log from `mpv` as well? (`mpv -v https://www.vimeo.com/240640253`)

**Comment by parkmino** [comment]
As attached

[log.txt](https://github.com/gnome-mpv/gnome-mpv/files/1436165/log.txt)

**Comment by gnome-mpv** [comment]
The version of `mpv` you're using (git-2017-10-28-6b74576) is much newer than the mpv library being used by `gnome-mpv` (0.26). Try updating the mpv library and see if that fixes the problem. On Debian derivatives, the package you need to update is `libmpv1`.

**Comment by parkmino** [comment]
Do you mean that the latest libmpv1 should be OK on which gnome-mpv depends?
OK and it'll take some time to check this.

Have a good day!

**Comment by gnome-mpv** [comment]
> Do you mean that the latest libmpv1 should be OK on which gnome-mpv depends?

Can you rephrase that? I'm not sure what you're saying.

**Comment by parkmino** [comment]
I think you said that the latest libmpv1 should solve this issue.
Am I misunderstood?

**Comment by gnome-mpv** [comment]
> I think you said that the latest libmpv1 should solve this issue.

Yes, updating `libmpv1` should fix the problem.

**Comment by parkmino** [comment]
OK, then I should wait till it is done by one of my ubuntu PPAs.

Thanks!
